src/pages/home/report/ReportActionItemMessage.tsx

```diff
@@ -77,10 +77,9 @@ function ReportActionItemMessage({action, displayAsGroup, reportID, style, isHid
                 fragment={fragment}
                 iouMessage={iouMessage}
                 isThreadParentMessage={ReportActionsUtils.isThreadParentMessage(action, reportID)}
-                attachmentInfo={action.attachmentInfo}
                 pendingAction={action.pendingAction}
-                source={(action.originalMessage as OriginalMessageAddComment['originalMessage'])?.source}
-                accountID={action.actorAccountID}
+                source={action.originalMessage as OriginalMessageSource}
```

Comment:
sorry forgot to post comment. Why `.source` is removed here?

Reply:
You might have confused with type. It's value.
Before: action.originalMessage?.source
After: action.originalMessage